### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by nikirago**
fixed install error.

### CODE REVIEW

File: app/_hub/yesinteractive/kongmap/index.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ configuration editing can be disabled by KongMap configuration, or managed with
 Full documentation is available here: [https://github.com/yesinteractive/kong-map/](https://github.com/yesinteractive/kong-map/)
 
 ## Compatibility ## 
-KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.5.
+KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.3 and supports both DB and Non-DB (dbless) Kong configurations. KongMap also supports Kong for Kubernetes Ingress Controller versions greater than 0.5 (In Kong for Kubernetes, the Ingress Controller's proxy container must have its Admin API exposed in some fashion.)
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:78]:
```suggestion
KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.3 and supports both Classic and DB-less Kong configurations. KongMap also supports Kong for Kubernetes Ingress Controller versions greater than 0.5. In Kong for Kubernetes, the Ingress Controller's proxy container must have its Admin API exposed in some fashion.
```
```suggestion
KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.3 and supports both DB and Non-DB (dbless) Kong configurations. KongMap also supports Kong for Kubernetes Ingress Controller versions greater than 0.5 (In Kong for Kubernetes, the Ingress Controller's proxy container must have its Admin API exposed in some fashion.)
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edit but @lena-larionova should have a look as well for current product terminology.

File: app/_hub/yesinteractive/kongmap/index.md (hunk 1 above)
Comment (lena-larionova) [reply app/_hub/yesinteractive/kongmap/index.md:78]: @tlblessing I'm assuming the second suggestion was an accident? In any case, I'd also suggest rephrasing the way we refer to the OSS and Enterprise versions:

```suggestion
KongMap supports Kong Gateway (OSS) and Kong Enterprise clusters version 1.5 and later, in both DB-backed and DB-less configurations. KongMap also supports Kong for Kubernetes Ingress Controller versions 0.6 and later. In Kong for Kubernetes, the Ingress Controller's proxy container must have its Admin API exposed in some fashion.
```
Ideally, we'd also link to docs telling users how to expose the Admin API but as far as I know, those docs don't exist.

Let's not refer to "Classic" here without any context.

@nikirago do "versions greater than 0.5" and "greater than version 1.3" mean 0.6 onward and 1.5 onward? It's confusing to refer to a version that isn't supported in this context.

Comment (tlblessing) [reply app/_hub/yesinteractive/kongmap/index.md:78]: @lena-larionova yeah, didn't intend for that second suggestion not sure what happened there. I like your suggested edits.

Comment (nikirago) [reply app/_hub/yesinteractive/kongmap/index.md:78]: @lena-larionova versions 0.5 onward and 1.3 and onward are supported.

**Review by nikirago — COMMENTED** [review]
moved install instructions to github.

**Comment by nikirago** [comment]
Had to change the kongmap icon in the hub per kong legal.

**Review by GayleNeumann — APPROVED** [review]
LGTM

**Pull request merged**